api/types/auxprogress/push.go

```diff
@@ -13,6 +14,8 @@ type ManifestPushedInsteadOfIndex struct {
 	// OriginalIndex is the descriptor of the original image index.
 	OriginalIndex ocispec.Descriptor `json:"originalIndex"`
 
+	ImageSummary *image.Summary
+
```

Comment:
Haven't given this a lot of thought yet, but wondering if this should be on the `ContentMissing` response; i.e., if we decide to not go ahead on this situation (image is incomplete, and multiple platforms to chose from), we would return the `ContentMissing`, with an overview of what's available.

w.r.t. presenting that to the user, I'm wondering if the tree view may be "too much info", or if a list of available platforms to choose from would be enough (slightly looking at the response requiring to collect / calculate all the bits, including sizes etc).